pybamm/solvers/idaklu_solver.py

```diff
@@ -97,12 +97,37 @@ def __init__(
         default_options = {
             "print_stats": False,
             "jacobian": "sparse",
-            "linear_solver": "SUNLinSol_KLU",
             "preconditioner": "BBDP",
-            "linsol_max_iterations": 5,
             "precon_half_bandwidth": 5,
             "precon_half_bandwidth_keep": 5,
             "num_threads": 1,
+            # IDA main solver
+            "max_order_bdf": 5,
+            "max_num_steps": 500,
+            # dt_init = 0.0 uses the default
+            "dt_init": 0.0,
+            # dt_max = 0.0 uses the default
+            "dt_max": 0.0,
```

Comment:
We should be able to make these comments on the same line to keep it from looking like commented out code
```suggestion
            "dt_init": 0.0,  # The solver default is used if this is left at zero
            "dt_max": 0.0,  # The solver default is used if this is left at zero
```